kolibri/logger/models.py

```diff
@@ -98,3 +100,56 @@ class UserSessionLog(BaseLogModel):
     start_timestamp = models.DateTimeField(auto_now_add=True)
     completion_timestamp = models.DateTimeField(blank=True, null=True)
     pages = models.TextField(blank=True)
+
+class MasteryLog(BaseLogModel):
+    """
+    This model provides a summary of a user's engagement with an assessment within a mastery level
+    """
+    # Every MasteryLog is related to the single summary log for the user/content pair
+    summarylog = models.ForeignKey(ContentSummaryLog, related_name="masterylogs")
+    # The MasteryLog records the mastery criterion that has been specified for the user.
+    # It is recorded here to prevent this changing in the middle of a user's engagement
+    # with an assessment.
+    mastery_criterion = models.CharField(max_length=200)
```

Comment:
Will this always be long enough? Might be better not to have a max length, and change it to a `TextField`?